setup.py

```diff
@@ -33,9 +33,14 @@ def get_extras_require():
         "dev-tools",
         "all-contrib-expectations",
     )
-    rx_fname_part = re.compile(r"requirements-dev-(.*).txt")
-    for fname in sorted(glob("requirements-dev-*.txt")):
-        key = rx_fname_part.match(fname).group(1)
+    requirements_dir = "reqs"
+    rx_fname_part = re.compile(rf"{requirements_dir}/requirements-dev-(.*).txt")
+    for fname in glob(f"{requirements_dir}/*.txt"):
```

Comment:
`pathlib` has `glob` & `match` methods as well.
And results are actually `Path` objects instead of strings.

I think it's fine to leave this as is though, just an FYI if you weren't already aware.

https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.match
https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob

Reply:
I know about `glob` but I didn't know about `match`!